src/cmd/usb-util.js

```diff
@@ -349,6 +349,10 @@ async function getOneUsbDevice({ idOrName, api, auth, ui, flashMode, platformId
 		devices = devices.filter(d => d.platformId === platformId);
 	}
 
+	// filter out linux kind devices
```

Comment:
Is there any command where we'd want getOneUsbDevice to return Linux devices? I don't think so right now, but it would be good to check the list of commands where this function is used.

Reply:
There are some places that currently should not be used for `linux` platforms:
`deviceProtection` (I'm not sure if in a future we're gonna have device protection on tachyons
`esim._changeLed`: Change led (this uses also sendControlRequest so tachyons might not work for this function
`keys.sendPublicKeyToServer`: requires DFU mode to work so `linux` devices might not work for this function
`serial.handleDeviceProtection`: Same here, linux devices might not ready for deviceProtection